src/Configuration/Definition/BaseExtractorConfigDefinition.php

```diff
@@ -58,11 +59,17 @@ protected function getSshNode(): ArrayNodeDefinition
                         ->scalarNode('public')->end()
                     ->end()
                 ->end()
-                ->scalarNode('sshHost')->end()
-                ->scalarNode('sshPort')->end()
+                ->scalarNode('sshHost')
+                    ->cannotBeEmpty()
+                ->end()
+                ->scalarNode('sshPort')
+                    ->defaultValue(22)
+                ->end()
                 ->scalarNode('remoteHost')->end()
                 ->scalarNode('remotePort')->end()
-                ->scalarNode('localPort')->end()
+                ->scalarNode('localPort')
+                    ->defaultValue(33006)
```

Comment:
tenhle default se vzal kde?

Reply:
Proč vlastně jde nastavovat? Co když dám špatný (80 nebo třeba nějaký na kterém něco poslouchá)? Není jedno jaký je? Nemohli bysme mít prostě "nějaký" napevno?

Reply:
Tohle je dalsi pozustatek, ktery respektuju. Ohledne konfigu je tam vice veci k doladeni, viz komentare od @tomasfejfar na povinnosti nekterych poli. Nic tam neni proto, ze bych to tak chtel, ale protoze to tak bylo, tak to prozatim zachovavam. Zalozim na to zvlast issue.

Reply:
no to byla, moje otazka - kde se to vzalo, takze for future reference: https://github.com/keboola/db-extractor-common/blob/master/src/Keboola/DbExtractor/Extractor/Extractor.php#L94 a z toho uz se da udelat blame

Reply:
a dal bych to ted neresil

Reply:
Takze sis ted vlastne odpovedel sam 😄

Reply:
jj, vubec jsem to predtim v tom starym kodu nenasel